lib/web_ui/dev/firefox.dart

```diff
@@ -43,13 +44,23 @@ class Firefox extends Browser {
         infoLog: isCirrus ? stdout : DevNull(),
       );
 
+      // Using a profile on opening will prevent popups related to profiles.
+      final _profile = '''
+user_pref("browser.shell.checkDefaultBrowser", false);
+user_pref("dom.disable_open_during_load", false);
+user_pref("dom.max_script_run_time", 0);
+''';
+
       // A good source of various Firefox Command Line options:
       // https://developer.mozilla.org/en-US/docs/Mozilla/Command_Line_Options#Browser
       //
       var dir = createTempDir();
+      File(path.join(dir, 'prefs.js')).writeAsStringSync(_profile);
```

Comment:
Should we clean up the profile directory between test runs?

Reply:
We delete the directory after processExits: 
Line 78: unawaited(process.exitCode
          .then((_) => Directory(dir).deleteSync(recursive: true)));